### DISCUSSION TITLE
v1.0.0-beta.4

### DISCUSSION THREAD

**Discussion by 4gray** [4gray/iptvnator#493] [General]
BlueSky: https://bsky.app/profile/iptvnator.bsky.social
Telegram: https://t.me/iptvnator
Support on Patreon: https://patreon.com/4gray 
Github sponsors: https://github.com/sponsors/4gray

---
🔥 Changes:
- [Xtream] Completely reworked xtream API feature 
  - Global search feature 
  - Recently viewed items (history)
  - Favorites list
  - New UI with improved navigation
  - Account info dialog
  - Improved mpv player integration (see what is playing now, close mpv instance direcly from iptvnator)
- [M3U] Multi-EPG view is back 
- [AppImage] Activate bundleMediaFramework in packaging configurations for AppImage
- [Settings] Fix save operation
- [Home] Ctrl/Cmd+F shortcut to focus search field

See the assets to download this version and install.

<hr /><em>This discussion was created from the release <a href='https://github.com/4gray/iptvnator/releases/tag/app-v1.0.0-4'>v1.0.0-beta.4</a>.</em>

**Comment by fuengfah** [comment]
Can I request PR to add HTTP User Agent?

**Reply by 4gray** [reply]
For m3u playlists? Sure, feel free to contribute.

**Comment by IsraelCrispim** [comment]
Hello, I'm running it through the Code, but it doesn't load when I run Tauri, it stays on the loading screen, but in the version released by the author it is running and the list works... Can someone help me solve this?

**Reply by 4gray** [reply]
Please check the logs

**Comment by spike1986** [comment]
![Bildschirmfoto vom 2024-12-17 11-54-39](https://github.com/user-attachments/assets/ef3c4193-2ab0-4932-931c-5b12f14960bc)
![Bildschirmfoto vom 2024-12-17 11-55-00](https://github.com/user-attachments/assets/1817d8a1-c6af-4046-8968-a0f563a53cc0)

Hi, I've switched to IPTVnator a few weeks ago, and I have to say, I'm impressed. I also tested the new Beta version. In LiveTV, the channels are no longer displayed in groups, but it works fine for VOD (see attachment). I integrated it using the XTREME Code.

I also have a question: Is there any way to display the EPG across multiple channels? I haven’t found this option so far. When I click on a single channel, the EPG shows up fine, though.

**Reply by glennlunder** [reply]
Same experience as @spike1986 here - somewhere between beta 3 and beta 4, the categories for Live TV disappeared, and that makes browsing very very hard for large playlists. My playlist is also an Xtreme code one.

**Reply by 4gray** [reply]
Working on it, categories will appear in the next beta. Can you please leave some feedback about OS and player that works for you? Thanks!

**Reply by glennlunder** [reply]
I am testing on Windows 11, using MPV as replayer (installed via Scoop).

**Reply by zkazsi** [reply]
I very much enjoy using iptvnator, and also very much would like to see the return of Live TV categories. I use Linux (arch)

Thanks for all your work

**Comment by kamilo-olimak** [comment]
hello Im running beta version 1.0.0.4 on a win10 and I can't enter fullscreen, I don't know if it us not possible or if Im using the app wrong?
thanks in advance
(btw congratulations! this is the best app so far I been tested for Microsoft windows environment)

**Reply by kamilo-olimak** [reply]
Also you can't play from Favorites, you can add channels to fav, you can browse fav list but is not working, you can't go in the channel from the fav list.

**Comment by AryaDarth** [comment]
export function does not work, clicking the button does nothing at all. Tested on this release for the Macos intel chip (iptvnator_1.0.0-4_x64.dmg)

**Comment by CokaCrack** [comment]
I use the AUR package on manjaro but it is only the version 1.0.0.2 . is there a plan to update it to the most recent version?

**Reply by glennlunder** [reply]
It is often not the case that the author of the software and the maintainer of a particular package in the AUR or anywhere else is the same. It seems there are two different versions of iptvnator currently on the AUR, see https://aur.archlinux.org/packages?O=0&SeB=nd&K=%C3%AFptvnator&outdated=&SB=p&SO=d&PP=50&submit=Go for more info on them and their maintainers.

If you want the absolute latest, I suggest just building it yourself.

**Reply by CokaCrack** [reply]
Ok did not know that. But I don't think i have the skill to do this so, look like I'll wait for it. Thanks

**Comment by hamzah077** [comment]
Hi 
Thank you for your amazing work 
Can i ask for 2 things it will make it better (Windows) 
1-  Add Resume play from where i stopped watching last time for movies and TV series  [Xtream] 
2- Add (Recently added ) for movies and TV series [Xtream] 
this app almost complete thanks again .

**Comment by jotenakis2** [comment]
Is there any rpm aarch64 build somewhere ? Or a flatpak for aarch64 ?
Thanks

**Comment by 4ddict** [comment]
I love using your software, but have to revert to another client whenever I am watching TV from the couch (I use a simple remote).

- Can you please add a feature to switch channels (a simple next/previous). :)
- A option to not have the video be full screen, but still have the video cover the entire application (i frequently use split screen, and would like it to only display the video).

**Comment by aftabkhaliq86** [comment]
i am getting this error  i just clone the repo 
nom install 
and npm run tauri dev
but getting this error 

![image](https://github.com/user-attachments/assets/01d71173-bedf-4789-9082-65b37275847b)
